jdaviz/configs/imviz/helper.py

```diff
@@ -79,6 +94,159 @@ def load_data(self, data, parser_reference=None, **kwargs):
             self.app.load_data(
                 data, parser_reference=parser_reference, **kwargs)
 
+    # astrowidgets ducktyping (MVP)
+
+    def center_on(self, point):
+        raise NotImplementedError
+
+    def offset_to(self, dx, dy, skycoord_offset=False):
+        raise NotImplementedError
+
+    @property
+    def zoom_level(self):
+        raise NotImplementedError
+
+    @zoom_level.setter
+    def zoom_level(self, val):
+        raise NotImplementedError
+
+    def zoom(self, val):
+        raise NotImplementedError
+
+    @property
+    def stretch_options(self):
+        raise NotImplementedError
+
+    @property
+    def stretch(self):
+        raise NotImplementedError
+
+    @stretch.setter
+    def stretch(self, val):
+        raise NotImplementedError
+
+    @property
+    def autocut_options(self):
+        raise NotImplementedError
+
+    @property
+    def cuts(self):
+        raise NotImplementedError
+
+    @cuts.setter
+    def cuts(self, val):
+        raise NotImplementedError
+
+    @property
+    def colormap_options(self):
+        raise NotImplementedError
+
+    def set_colormap(self, cmap):
+        raise NotImplementedError
+
+    @property
+    def click_center(self):
+        return False  # No way to do this right now
+
+    @click_center.setter
+    def click_center(self, val):
+        raise NotImplementedError
+
+    @property
+    def click_drag(self):
+        return False  # No way to do this right now
+
+    @click_drag.setter
+    def click_drag(self, value):
+        raise NotImplementedError
+
+    @property
+    def scroll_pan(self):
+        return False  # No way to do this right now
+
+    @scroll_pan.setter
+    def scroll_pan(self, value):
+        raise NotImplementedError
+
+    def save(self, filename):
+        raise NotImplementedError
+
+    # markers (MVP)
```

Comment:
@eteq -- Are we on the same page w.r.t. markers now?

Still need to discuss loaders.

Reply:
Markers MVP: See #699